webpack.config.js

```diff
@@ -88,7 +88,7 @@ export let webConfig = Object.assign({}, defaultWebpackConfig, {
         pathinfo: false
     },
     plugins: [
-        // new webpack.optimize.ModuleConcatenationPlugin(),
+        new webpack.optimize.ModuleConcatenationPlugin(),
```

Comment:
When this was enabled in `paypal-checkout` I saw errors when I loaded `checkout.js` in a browser. Can you please take a look and see what was going wrong, and if the same problem happens here?